obspy/io/nlloc/__init__.py

```diff
@@ -15,11 +15,47 @@
 
 Example
 -------
+If NonLinLoc location run was performed using one of the standard projections
+available in the NonLinLoc package, it is straightforward to read a NonLinLoc
+Hypocenter-Phase file into an ObsPy :class:`~obspy.core.event.Catalog` object:
 
-First, if NonLinLoc location run was performed in some custom coordinate system
-(as opposed to WGS84 with depth in meters down), we need to set up a coordinate
-conversion function to convert from the NonLinLoc location coordinates `x`,
-`y`, `z` to longitude, latitude and depth in kilometers.
+>>> from obspy import read_events
+>>> cat = read_events("/path/to/nlloc.hyp")
+>>> print(cat)
+1 Event(s) in Catalog:
+2006-07-15T17:21:20.195670Z |  +7.737,  +51.658
+
+>>> event = cat[0]  # doctest:
+>>> print(event)  # doctest: +ELLIPSIS +NORMALIZE_WHITESPACE
+Event:	2006-07-15T17:21:20.195670Z |  +7.737,  +51.658
+<BLANKLINE>
+       resource_id: ResourceIdentifier(id="smi:local/...")
+     creation_info: CreationInfo(creation_time=UTCDateTime(2013, 6, 21,
+12, 53, 34), version='NLLoc:v6.02.07')
```

Comment:
Since you have whitespace normalization on, you might as well align this one properly.